tests/e2e/high_availability/coord_cluster_registration.py

```diff
@@ -40,7 +40,7 @@ def get_instances_description():
                 "--management-port",
                 "10011",
             ],
-            "log_file": "instance_1.log",
+            "log_file": "high_availability/coord_cluster_registration/instance_1.log",
```

Comment:
Can you put this as some kind of constant in the file

Reply:
coord_cluster_registration as constant

Reply:
I think it is okay like this